src/panels/config/logs/dialog-system-log-detail.ts

```diff
@@ -66,13 +73,27 @@ class DialogSystemLogDetail extends LitElement {
         opened
         @opened-changed="${this._openedChanged}"
       >
-        <h2>
-          ${this.hass.localize(
-            "ui.panel.config.logs.details",
-            "level",
-            item.level
-          )}
-        </h2>
+        <div class="heading">
+          <h2>
+            ${this.hass.localize(
+              "ui.panel.config.logs.details",
+              "level",
+              item.level
+            )}
+          </h2>
+          <ha-svg-icon
+            id="copy"
+            .path=${mdiContentCopy}
+            @click=${this._copyLog}
+          ></ha-svg-icon>
```

Comment:
```suggestion
          <mwc-icon-button id="copy" @click=${this._copyLog}>
            <ha-svg-icon .path=${mdiContentCopy}></ha-svg-icon>
          </mwc-icon-button>
```

Reply:
`<ha-icon-button>`?

Reply:
No, that should only be used if the icon is user-configurable.